src/ui/backend/canvas/ViewBacking.js

```diff
@@ -169,19 +195,17 @@ var ViewBacking = exports = Class(BaseBacking, function () {
 			this._view.render && this._view.render(ctx, opts);
```

Comment:
Oops, you're right, we would need to put this in a separate function! That would work :)